src/env.js

```diff
@@ -12,6 +12,7 @@
     window.__env.natsServerUrl = 'https://identityevents-dev.energyweb.org/';
     window.__env.kmsServerUrl = undefined;
     window.__env.stakingPoolFactoryAddress = '0xd61Ac65c0fECfb83108f6E374a57FfF9c692901d';
+    window.__env.claimManagerAddress = '';
```

Comment:
This is the address for `dev`. How does this get configured for `staging`/`volta`
```suggestion
    window.__env.claimManagerAddress = '0xC3dD7ED75779b33F5Cfb709E0aB02b71fbFA3210';
```

Reply:
It should be done by devops team -> i need to add description how to handle that. Otherwise if we change a contract -> we need to rebuild and redeploy entire switchboard. With above you just change property in env.js file. Read this one as i mentioned above: https://www.jvandemo.com/how-to-use-environment-variables-to-configure-your-angular-application-without-a-rebuild/

Reply:
ClaimManager addresses:
`dev` : `0xC3dD7ED75779b33F5Cfb709E0aB02b71fbFA3210`
`staging`: `0x561785174DF7f564f2591bA52B253c0F663427aB`